app/controllers/posts_controller.rb

```diff
@@ -4,8 +4,13 @@ class PostsController < ApplicationController
 
   def create
     @post = current_user.posts.create(post_params)
-    # redirect_to posts_url
-    redirect_to posts_url
+
+    if (@post.wall_id == 0)
+      redirect_to posts_url
+    else
+      @user_url = '/users/' + @post.wall_id.to_s
```

Comment:
.. hey Rob, dyou mean to use an '_assignment operator_' = or an 'equality operator' == here?

Reply:
We mean assignment operator.
If post.wall_id == 0 # this is for newsfeed posts - where the wall number is 0
else
#set the url to be the wall owner's url

Reply:
thanks Rob